src/main/kotlin/no/nav/familie/pdf/pdf/TabellUtils.kt

```diff
@@ -11,68 +11,73 @@ import com.itextpdf.layout.properties.UnitValue
-                    Paragraph(caption).apply {
+                    Paragraph(tabellData.label).apply {
                         setFontColor(DeviceRgb(0, 52, 125))
                         setFontSize(14f)
                         simulateBold()
                     },
                 )
             }
         tabell.caption = captionDiv
-
         tabell.addCell(lagTabellOverskriftscelle("Spørsmål"))
         tabell.addCell(lagTabellOverskriftscelle("Svar", false))
-        lagTabellRekursivt(tabellData, tabell)
+        tabell.lagTabellRekursivt(tabellData.verdiliste)
         return tabell
     }
 
-    private fun lagListeMedAlleElementer(
-        elementer: List<VerdilisteElement>,
-        strengManSkalSplitteTabellPå: String,
-    ): List<List<VerdilisteElement>> {
-        val listeMedAlleElementer = mutableListOf<List<VerdilisteElement>>()
-        var nåværendeElement = mutableListOf<VerdilisteElement>()
-        elementer.forEachIndexed { index, item ->
-            if (item.label == strengManSkalSplitteTabellPå && index != 0) {
-                listeMedAlleElementer.add(nåværendeElement)
-                nåværendeElement = mutableListOf()
-            }
-            nåværendeElement.add(item)
+    private fun lagTabellOverskriftscelle(
+        tekst: String,
+        erVenstreKolonne: Boolean = true,
+    ): Cell =
+        Cell().apply {
+            add(
+                Paragraph(tekst).apply {
+                    setFontColor(DeviceRgb(0, 86, 180))
+                    setFontSize(14f)
+                    simulateBold()
+                },
+            )
+            setBorder(Border.NO_BORDER)
+            if (erVenstreKolonne) setPaddingRight(10f) else setPaddingLeft(10f)
+            accessibilityProperties.role = StandardRoles.TH
         }
-        listeMedAlleElementer.add(nåværendeElement)
-        return listeMedAlleElementer
-    }
 
-    private fun lagTabellRekursivt(
+    private fun Table.lagTabellRekursivt(
         tabellData: List<VerdilisteElement>,
-        tabell: Table,
     ) {
-        tabellData.forEach { item ->
-            val label = item.label
-            val value = item.verdi ?: ""
+        tabellData.forEach { element ->
+            val verdi = element.verdi ?: ""
             when {
-                item.verdi != null -> {
-                    tabell.addCell(lagTabellInformasjonscelle(label, erUthevet = true))
-                    tabell.addCell(lagTabellInformasjonscelle(value.ifEmpty { " " }, false))
-                }
-
-                item.verdiliste != null -> {
-                    lagTabellRekursivt(item.verdiliste, tabell)
+                element.verdi != null -> {
+                    this.addCell(lagTabellInformasjonscelle(element.label, erUthevet = true))
+                    this.addCell(lagTabellInformasjonscelle(verdi.ifEmpty { " " }, erVenstreKolonne = false))
                 }
+                element.verdiliste != null -> lagTabellRekursivt(element.verdiliste)
```

Comment:
Sjekk opp om dette faktisk er rekursjon og evt. sjekk opp logikk/navngivning

Reply:
Vi trenger fortsatt denne. Barn har for eksempel flere under-objekter, son Terminbekreftelse, Barnets andre forelder og under Barnets andre forelder igjen har vi Persondata bare i vår `søknad.json`